website/source/docs/acl/acl-identity-providers.html.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: "docs"
+page_title: "ACL Identity Providers"
+sidebar_current: "docs-acl-identity-providers"
+description: |-
+  An ACL Identity Provider is a trusted external party that can be configured to to allow applications to authenticate to Consul using existing credentials and receive ACL Tokens usable within the local datacenter in exchange.
+---
+
+-> **1.5.0+:**  This guide only applies in Consul versions 1.5.0 and later.
+
+# ACL Identity Providers
+
+An ACL Identity Provider is a trusted external party that can be configured to
+to allow applications to authenticate to Consul using existing credentials and
+receive ACL Tokens usable within the local datacenter in exchange.
+
+The only supported type of identity provider in Consul 1.5 is `kubernetes`
+but it is expected that more will come later.
```

Comment:
I tend to think we should leave out any future plans/roadmap language in the docs and guides.